spec/presenters/cancellation_presenter_spec.rb

```diff
@@ -5,20 +5,11 @@
   let(:good_url_with_path) { 'http://example.com/asdf?qwerty=123' }
   let(:bad_url) { 'http://evil.com/asdf/qwerty' }
 
-  let(:view_context) { ActionView::Base.new }
```

Comment:
```
DEPRECATION WARNING: ActionView::Base instances should be constructed with a lookup context, assignments, and a controller.
```

The presenter class only uses the view_context to access the request referer header, so I slightly refactored to pass that in directly instead of using the view context.